ComprefaceTestApp/Configuration/ComprefaceConfiguration.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace ComprefaceTestApp.Configuration;
+
+public class ComprefaceConfiguration
```

Comment:
Convert to interface and create 2 implementations - 1st one from config file (name should be provided). Probably passing IConfiguration will work as well i.e. to find a section etc. Raise exception if parameters are not set
2nd one should set parameters explicitly and injected as well.

Reply:
Done